vocabularies/voc4cat/0007120.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007120>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007120"^^xsd:token ;
+    skos:altLabel "linear scaling calibration; proportional calibration; slope adjustment"@en ;
```

Comment:
The 2nd and 3rd alternate labels could be dropped. They are not exactly the same.